app/index.html

```diff
@@ -27,18 +27,18 @@
 				alert('login, here I come!');
 			}
 	};
-	
+
 	window.parent.kmc.vars = {
 
-		ks : 'YOUR_KS',
-		partner_id : 'YOUR_PID',
-		service_url : 'YOUR_SERVICE_URL',
-		cdn_host : 'YOUR_CDN_HOST',
-		liveanalytics : {player_id : 'YOUR_PLAYER_UICONF_ID'}
+		ks : 'djJ8MTc2MzMyMXy1PmA3FtA4jIUlDKrGivuTVPvHXOVcAbsqefa1_5-veLaA4IHmHT-Aq_gq09nQwk6bvGJCG8RYWOBu38pvv_kGO80F8oAUabKtvvLhoClkYXmzG643OZhUWxbeG1lFkz0k0SfaXAZCY1NuR2GaBg_8',
```

Comment:
LOOOOOOO!!!
please return the placeholders instead of actual data, and make sure to never commit real data here.